hummingbot/client/liquidity_bounty/terms_and_conditions.txt

```diff
@@ -3,30 +3,72 @@
 
-
 Data Sharing
-By participating in a Bounty Program, you are agreeing to sharing the following information with CoinAlpha, the Token Issuer that is conducting the Bounty Program, and their respective affiliates:
+By participating in a Bounty Program, you are agreeing to sharing the following information with CoinAlpha and the Sponsor that is conducting the Bounty Program, and their respective affiliates:
 
-Orders created by your use of Hummingbot;
-Transactions completed by your use of Hummingbot;
-Your Ethereum wallet address: where bounty awards, if any, are to be received.
+- Ethereum wallet address
+- Orders created: base currency, quote currency, and size
+- Orders filled: base currency, quote currency, and size
+- Exchange
+- Template or strategy names
+- Commands entered in the client interface
+- Log entries for errors, notifications, and trade events
+- Client ID: a UUID generated by Hummingbot when first configured, which can also be manually entered by the User
 
 When sharing data from Hummingbot, you are solely responsible for configuring what information you agree to share through the Hummingbot log configuration settings (please see example log templates).
 
+Data Monitoring
+It is the policy of CoinAlpha to prohibit and prevent market manipulation and unlawful trading activities. Users are prohibited from using Hummingbot for purposes of market manipulation, including but not limited to wash trading, spoofing, pump-and-dump schemes or any other practices intended to mislead others by artificially affecting market activity.
+In the course of providing the Services, CoinAlpha may monitor and detect “suspicious activity” in the form of potential market manipulation, including activity conducted by Hummingbot accounts that have opted-in to liquidity bounty programs and associated data collection. Based on any “suspicious activity" data, CoinAlpha may determine whether any additional steps are required, including reporting this activity to the Client, which may include the Sponsor or exchange(s). Throughout the process CoinAlpha may document when and how this monitoring is carried out and the rationale for handling the activity.
+CoinAlpha reserves the right, at its sole discretion, to share user-provided information and analysis, and reports about potential market manipulation to the Sponsors. By participating in Bounty Programs, you consent to such sharing, acknowledge that it may have an impact on your accounts and activity, disclaim any potential liability against CoinAlpha or Hummingbot, and waive any potential claims against CoinAlpha or Hummingbot.
 
 For additional terms related to the sharing and use of your data, please review the following:
+- CoinAlpha’s Privacy Policy
```

Comment:
For these, can you add the links after the text?

Reply:
done